### PR TITLE
PRL-5661: Routes clean up

### PR DESCRIPTION
**Pull request by hcarslaw**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/PRL-5661

### Change description ###
Cleans up the routes file

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/steps/testing-support/get.ts [hunk 1]
```diff
@@ -1,9 +0,0 @@
-import { GetController } from '../../app/controller/GetController';
```
Comment (vivek-sekhar) [inline src/main/steps/testing-support/get.ts:1]: chk all testing support files that are deleted with @sealDebolina

**Review by vivek-sekhar — CHANGES_REQUESTED** [review]

File: src/main/steps/c100-rebuild/start/getController.ts [hunk 2]
```diff
@@ -0,0 +1,45 @@
+import autobind from 'autobind-decorator';
+import { Response } from 'express';
+
+import { CaseWithId, FieldPrefix } from '../../../app/case/case';
+import { AppRequest } from '../../../app/controller/AppRequest';
+import { GetController, TranslationFn } from '../../../app/controller/GetController';
+import { C100_CHILD_ADDRESS } from '../../urls';
+
+@autobind
+export default class C100StartGetController extends GetController {
```
[the remark this answers is not in the document]
Comment (hcarslaw) [reply src/main/steps/c100-rebuild/start/getController.ts]: Changed to use post controller

File: src/main/app/controller/PostController.ts [hunk 3]
```diff
@@ -44,24 +36,6 @@ export class PostController<T extends AnyObject> {
...
-  private async saveBeforeSessionTimeout(req: AppRequest<T>, res: Response, formData: Partial<Case>): Promise<void> {
-    try {
-      await this.save(req, formData, this.getEventName(req));
-    } catch {
-      // ignore
-    }
-    res.end();
-  }
-
   private async saveAndContinue(req: AppRequest<T>, res: Response, form: Form, formData: Partial<Case>): Promise<void> {
```
[the remark this answers is not in the document]
Comment (hcarslaw) [reply src/main/app/controller/PostController.ts:38]: removed

File: src/main/app/controller/RespondentSubmitResponseController.ts [hunk 4]
```diff
@@ -13,23 +12,6 @@ const UID_LENGTH = 36;
...
-    const client = new CosApiClient(req.session.user.accessToken, req.locals.logger);
-    const caseData = toApiFormat(req?.session?.userCase);
-
-    const updatedCaseDataFromCos = await client.submitRespondentResponse(caseReference, partyId, caseData);
-    Object.assign(req.session.userCase, updatedCaseDataFromCos);
-
-    req.session.save(() => res.redirect(CA_RESPONDENT_RESPONSE_CONFIRMATION));
-  }
-
   public async getDraftDocument(req: AppRequest, res: Response): Promise<void> {
```
[the remark this answers is not in the document]
Comment (vivek-sekhar) [reply src/main/app/controller/RespondentSubmitResponseController.ts:15]: You can use download doc CC for downloading and create the doc in the controller

**Review by vivek-sekhar — CHANGES_REQUESTED** [review]

**Review by vivek-sekhar — APPROVED** [review]

**Pull request merged**